content/docs/delegators/_index.md

```diff
@@ -8,4 +8,4 @@ description: >
 
 Staking is the process of locking up _Newton_ tokens to help secure the network.  In return, the staker (or delegator) earns a proportion of the network transaction fees paid by users of the blockchain.  See the [Staking](/concepts/staking/) and [System model](/concepts/system-model/) sections of this document for further information about the role of staking in the Autonity network.
 
-The steps in this section describe how to carry out operations related to staking.  It is assumed that you the reader has setup the `aut` CLI tool, has an [account](/account-holders/create-acct/) with [fund](/account-holders/fund-acct/) and is able to submit transactions to the network.  See the [Account Holder Guide](/account-holders/) for details.
+The steps in this section describe how to carry out operations related to staking.  It is assumed that you the reader have setup the `aut` CLI tool, has an [account](/account-holders/create-acct/) with [fund](/account-holders/fund-acct/) and is able to submit transactions to the network.  See the [Account Holder Guide](/account-holders/) for details.
```

Comment:
"you the reader".. can't we just say "you"?

Reply:
Edited here and in:

./content/docs/developer/_index.md
./content/docs/node-operators/_index.md